package.json

```diff
@@ -33,13 +33,13 @@
     "devexpress-diagram": "2.1.39",
     "devexpress-gantt": "3.0.19",
     "devextreme-quill": "~1.4.2",
+    "devextreme-showdown": "^1.0.1",
```

Comment:
Is this required for fixing CI? Or how does it connected with internal-tools bump?